src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -114,7 +114,7 @@ private fun mapMetaDataToColumnSql(metadataOption: MetadataOption): String {
         MetadataOption.ORIGIN_IDS -> ",${ORIGIN_ID.name}"
         // TODO should be just last_write with comma prefix after empty rows are eliminated https://jira.openlattice.com/browse/LATTICE-2254
         MetadataOption.LAST_WRITE -> ",max(${LAST_WRITE.name}) AS ${mapMetaDataToColumnName(metadataOption)}"
-        MetadataOption.ENTITY_KEY_IDS -> ",${ENTITY_KEY_IDS_COL.name}"
+        MetadataOption.ENTITY_KEY_IDS -> ",array_agg(${ORIGIN_ID.name}) as ${ENTITY_KEY_IDS_COL.name})"
```

Comment:
This won't work unless we always select origin_id in inner query. I think it should be ",array_agg(${ENTITY_KEY_IDS_COL.name}) as ${ENTITY_KEY_IDS_COL.name})"

Reply:
It will work since it will be in inner query if metadata option is provided.